proposals/cloudedgeclientCG/charter.md

```diff
@@ -25,21 +25,21 @@ The following are out of scope:
 
 The Group will have succeeded if it can achieve the following:
 
-* Participation from various stakeholder communities, especially from startup, innovators in the space of distributed computing;
-* A standard solution architecture for workload offloading.
+* Participation from various stakeholder communities, especially from startups and innovators in the space of distributed computing;
+* Consensus on a standard solution architecture for workload offloading.
 
 ## Deliverables
 
 ### Specifications
 No Specifications will be produced under the current charter.
 
 ### Non-Normative Reports
-The group may produce other Community Group Reports within the scope of this charter but that are not Specifications, for instance use cases, requirements, or white papers.
+The group may produce other Community Group Reports within the scope of this charter that are not Specifications, for instance the group may produce use cases, requirements, or white papers.
```

Comment:
Looks good!
We could also explicitly mention studying ethical, privacy and security aspects.